dagshub/data_engine/datasources.py

```diff
@@ -169,7 +169,7 @@ def get_datasources(repo: str) -> List[Datasource]:
 
 
 def get_from_mlflow(
-    run: Optional[Union["mlflow.entities.Run", str]] = None, artifact_name=DEFAULT_MLFLOW_ARTIFACT_NAME
+    run: Optional[Union["mlflow.entities.Run", str]] = None, artifact_name: Optional[str] = None
```

Comment:
It is loading the datasource + query + assigned dataset (if there was any)
It WILL NOT load the time of the query as a global as_of. I guess we can add a flag to alter this behavior and either load with the timestamp or not.